installer/bootenv.go

```diff
@@ -129,6 +154,30 @@ func (e *UBootEnv) WriteEnv(vars BootVars) error {
 		log.Errorln("fw_setenv returned failure: ", err)
 		return err
 	}
+	return e.validate(vars)
+}
+
+// validate validates the given UBoot environment variables in 'vars', by
+// returning the UBoot environment and comparing them with the expected
+// environment.
+func (e *UBootEnv) validate(expected BootVars) error {
+	if e.SkipValidate {
```

Comment:
I know this is not the prettiest solution, but it is certainly the least intrusive in terms of how much code that needs to change in order to support overriding this one method in the tests (composition over inheritance isn't always a treat).